nbs/models.ipynb

```diff
@@ -26,7 +26,7 @@
    "source": [
```

Comment:
<p>SimpleExponentialSmoothing? But SimpleSmooth is shorter.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Yaaa, for the moment I added a shorter name for it to fit in the index.
In the future we can explore increasing the index size.